src/WasmSdk/Sdk/Sdk.props

```diff
@@ -25,6 +25,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- JS Modules - We disable the manifest generation because we are going to inline the modules in the blazor.boot.json manifest -->
     <GenerateJSModuleManifest>false</GenerateJSModuleManifest>
 
+    <!-- Static web assets defaults -->
+    <StaticWebAssetMakeReferencedAssetOriginalItemSpecAbsolute Condition="'$(StaticWebAssetMakeReferencedAssetOriginalItemSpecAbsolute)' == ''">true</StaticWebAssetMakeReferencedAssetOriginalItemSpecAbsolute>
+
```

Comment:
There are similar properties in the Blazor WebAssembly SDK, we should remove them from there since that SDK imports this SDK and they are no longer needed